.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "vets-api native setup",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/ruby-asdf:0": {
+			"version": "3.2.3"
+		},
+		"ghcr.io/robbert229/devcontainer-features/postgresql-client:1": {
+			"version": "15"
+		},
+		"ghcr.io/devcontainers-contrib/features/redis-homebrew:1": {
+			"version": "6.2"
+		}
+	},
+
+	"forwardPorts": [
+		3000,
+		9293,
+		5432,
+		6379
+	],
+	"portsAttributes": {
+		"3000": {
+			"label": "vets-api",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		},
+		"9293": {
+			"label": "vets-api-healthcheck",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		},
+		"5432": {
+			"label": "postgis",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		},
+		"6379": {
+			"label": "redis",
+			"onAutoForward": "notify",
```

Comment:
I'm not sure if we need notify on all these but port 3000 seems handy.

Reply:
Makes sense, changing